scripts/recall-last-item.lua

```diff
@@ -1,18 +1,34 @@
 local util = require("scripts.util")
 
+local MAX_HISTORY_SIZE = 10
```

Comment:
I don't usually capitalize constants in Lua, so please change this to `max_history_size`.

Reply:
fixed